enforcer/src/main/java/org/wso2/micro/gateway/enforcer/security/jwt/issuer/AbstractJWTIssuer.java

```diff
@@ -78,7 +79,7 @@ public String getDialectURI() {
     }
 
     public String encode(byte[] stringToBeEncoded) throws JWTGeneratorException {
-        return java.util.Base64.getUrlEncoder().encodeToString(stringToBeEncoded);
+        return Base64.getUrlEncoder().withoutPadding().encodeToString(stringToBeEncoded);
```

Comment:
Won't it be a good idea to use a library like [nimbus](https://bitbucket.org/connect2id/nimbus-jose-jwt) to generate the JWT. Would help avoid similar issues in future.